src/components/Editor/GutterMenu.js

```diff
@@ -154,19 +153,16 @@ class GutterContextMenuComponent extends PureComponent {
   }
 }
 
-export default connect(
-  state => {
-    const selectedSource = getSelectedSource(state);
-    return {
-      selectedLocation: getSelectedLocation(state),
-      selectedSource: selectedSource,
-      breakpoints: getVisibleBreakpoints(state),
-      pauseData: getPause(state),
-      contextMenu: getContextMenu(state),
-      emptyLines: selectedSource
-        ? getEmptyLines(state, selectedSource.toJS())
-        : []
-    };
-  },
-  dispatch => bindActionCreators(actions, dispatch)
-)(GutterContextMenuComponent);
+export default connect(state => {
```

Comment:
This seems very hard to read, maybe we just make a mapStateToProps and mapDispatchToProps function to handle the logic. 1.) It is the default way of handling these kinds of things and 2.) will make it easier to see what is going on and also 3.) eliminate mistakes due to nested bracketing.

Reply:
I'd like to figure out how to have `connect(mapStateToProps, actions)(GutterContextMenuComponent);` everywhere

`